gcm/gcm.py

```diff
@@ -263,79 +266,49 @@ def extract_unsent_reg_ids(self, info):
             return info['errors']['Unavailable']
         return []
 
-    def plaintext_request(self, registration_id, data=None, collapse_key=None,
-                          delay_while_idle=False, time_to_live=None, retries=5,
-                          dry_run=False, restricted_package_name=None, priority=None):
+    def plaintext_request(self, **kwargs):
         """
         Makes a plaintext request to GCM servers
 
         :param registration_id: string of the registration id
         :param data: dict mapping of key-value pairs of messages
         :return dict of response body from Google including multicast_id, success, failure, canonical_ids, etc
-        :raises GCMMissingRegistrationException: if registration_id is not provided
         """
 
-        if not registration_id:
```

Comment:
If we remove the checking on the client I think we should handle the check in raise_error method when the response comes back.